app/decorators/service_decorator.rb

```diff
@@ -45,7 +45,15 @@ def api_selector_api_link
     end
   end
 
-  alias link api_selector_api_link
+  def used_by_backend_table_data
```

Comment:
The name of this method almost sounds like a comment, a comment that would be slightly inaccurate if taken literally about what the methods is used by 🙂

Reply:
What do you suggest instead?

Reply:
IDK, but a service (API product) is not used _by_ a backend. Regardless, the response seems to be nothing related to backend. It's name, system_name and URL of the service, as JSON.

Reply:
Maybe just 

```suggestion
  def table_data
```

Reply:
Fair enough